lib/api-client.js

```diff
@@ -32,7 +32,7 @@ SlackAPIClient.prototype.SlackError = SlackError;
  * @param {Function} [callback] An optional callback to propagate the response to.
  * @callback {[Error, ResponseBody, Response]}
  */
-SlackAPIClient.prototype.api = function(method, options, callback) {
```

Comment:
Ah wait no, I forgot call is already taken by JavaScript https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call

.send() is clear enough.